AutorunsToWinEventLog/Install.ps1

```diff
@@ -9,7 +9,7 @@ $autorunsPath = "c:\Program Files\AutorunsToWinEventLog\Autorunsc64.exe"
 if(!(test-path $autorunsPath)) {
   # Requires TLS 1.2
   [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-  Invoke-WebRequest -Uri "https://live.sysinternals.com/autorunsc64.exe" -OutFile "$autorunsPath"
+  Invoke-WebRequest -Uri "http://live.sysinternals.com/autorunsc64.exe" -OutFile "$autorunsPath"
```

Comment:
My suggestion would be changing this to:

```
Try { 
  Invoke-WebRequest -Uri "https://live.sysinternals.com/autorunsc64.exe" -OutFile "$autorunsPath"
} Catch {
  Invoke-WebRequest -Uri "http://live.sysinternals.com/autorunsc64.exe" -OutFile "$autorunsPath"
}
```

Reply:
Great suggestion, definitely the way to go. Made the change!